Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -106,6 +112,34 @@ class HomeFragment : Fragment(R.layout.home_fragment_layout), AutoInject {
         binding.container.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
     }
 
+    private fun menuIconWithText(drawable: Drawable?, title: CharSequence): CharSequence {
+        if (drawable == null) return title
+        return SpannableString("    $title").apply {
+            drawable.setBounds(0, 0, drawable.intrinsicWidth, drawable.intrinsicHeight)
+            setSpan(ImageSpan(drawable, ImageSpan.ALIGN_BOTTOM), 0, 1, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+        }
+    }
```

Comment:
Why do we need to create `SpannableString` ? I see that you already added icons for menu item in xml

Reply:
The reason for this hack is that menu icons are used only for action items, not in the overflow menu